orchagent/pfcwdorch.cpp

```diff
@@ -400,24 +401,33 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
         return;
     }
 
+    set<uint8_t> losslessTc;
+    uint8_t pfcMask = attr.value.u8;
+    for (uint8_t i = 0; i < PFC_WD_TC_MAX; i++)
+    {
+        if ((pfcMask & (1 << i)) == 0)
+        {
+            continue;
+        }
+
+        losslessTc.insert(i);
+    }
+
     if (!c_portStatIds.empty())
     {
         string key = getFlexCounterTableKey(sai_serialize_object_id(port.m_port_id));
         vector<FieldValueTuple> fieldValues;
+        // Only register lossless tc counters in database.
         string str = counterIdsToStr(c_portStatIds, &sai_serialize_port_stat);
-        fieldValues.emplace_back(PORT_COUNTER_ID_LIST, str);
+        string losslessStr = getLosslessStr(str,losslessTc);
```

Comment:
please add space after the comma, make sure you are consistent in the code style.